packages/midiomatic/PKGBUILD

```diff
@@ -22,7 +22,7 @@ sha256sums=('aed203bde8fd01e279e058d48f084754f50b9fd758d7b56b845f11d8fbdbb1f3')
 
 build() {
   cd "$pkgname-$pkgver"
-  make
+  make -j1
```

Comment:
Interesting that the `MAKEFLAGS` cause an issue here, but maybe something I should have expected. But I do like this approach, use `-j4` by default and only add `-j1` for packages that don't support parallel builds.

Reply:
This is an issue with midiomatic, which I will fix in the next release.

But it hightlights an issue with our CI. We probably need a job that rebuilds all packages regularly to make sure the tooling didn't break the build process of any packages.